README.adoc

```diff
@@ -480,6 +480,47 @@ _ { raise_exception }.must_raise TypeError
 
 Check the http://docs.seattlerb.org/minitest/Minitest/Expectations.html[Minitest::Expectations doc] for more information about its usage.
 
+=== Hooks [[hooks]]
```

Comment:
It may be even clearer :-)

```suggestion
=== Lifecycle Hooks [[lifecycle-hooks]]
```

Reply:
I considered that, but the Minitest docs seem to indicate that 'lifecycle hooks' are things such as `before_setup` and `after_teardown`, which are "meant for library writers, NOT for regular test authors."

https://www.rubydoc.info/gems/minitest/5.9.1/Minitest/Test/LifecycleHooks

Reply:
I get it. Thank you for the explanation!